Source/DayView.swift

```diff
@@ -60,7 +60,22 @@ public final class DayView: UIView, TimelinePagerViewDelegate {
     }
   }
 
-  public var calendar: Calendar = Calendar.autoupdatingCurrent
+  public var calendar: Calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // TODO: Should we observe calendar in state rather then change it manualy?
+      dayHeaderView.calendar = self.calendar
+      timelinePagerView.calendar = self.calendar
+
+      let date = self.state?.selectedDate ?? Date()
+      let newState = DayViewState(date: date, calendar: calendar)
+      newState.move(to: date)
+      state = newState
+      
+      // TODO: Shound we redraw dayView and its subvies? Or we should do this when calendar in state was changed?
+//      setNeedsDisplay()
+//      self.subviews.forEach { $0.setNeedsDisplay() }
```

Comment:
Not sure whether the call should happen here, or at the respective `subView`. I'd recommend the latter. Think of using some of the subviews on their own, without being shown in the `DayView`.

Reply:
I agree with the same opinion. It seems to me that subviews redrawing must be triggered by a change in its state.

Reply:
Yeah, a state or a `calendar` property, depending on which way you decide to go.